packages/core/components/DirectoryTree/findChildNodes.ts

```diff
@@ -85,16 +95,13 @@ export async function findChildNodes(params: FindChildNodesParams): Promise<stri
     }
 
     const filteredValuesSorted = filteredValues.sort(naturalComparator);
-    if (!shouldShowNullGroups || userSelectedFiltersForCurrentAnnotation.length) {
+    // Don't add NO_VALUE_NODE if there are user-applied filters for the annotation
+    if (
+        !shouldShowNullGroups ||
```

Comment:
This is a nit more than anything but I think this is more readable in reverse ex. `shouldShowNullGroups && noValueFileCount > 0 && !userSelectedFiltersForCurrentAnnotation.length` but nbd either way. lgtm